chunky/src/java/se/llbit/chunky/renderer/scene/Sun.java

```diff
@@ -137,6 +137,10 @@ public class Sun implements JsonSerializable {
 
   protected final Vector3 emittance = new Vector3(1, 1, 1);
 
+  private double pE = FastMath.pow(DEFAULT_INTENSITY, Scene.DEFAULT_GAMMA);
+
+  protected final Vector3 previewEmittance = new Vector3(pE, pE, pE);
```

Comment:
```suggestion
  private static final double pE = FastMath.pow(DEFAULT_INTENSITY, Scene.DEFAULT_GAMMA);

  protected static final Vector3 previewEmittance = new Vector3(pE, pE, pE);
```

Reply:
does this even make sense? it may actually make more sense for these to be configurable for each sun instance :thinking:

certainly can add final to `pE` though

Reply:
I don't know if it makes sense, but I'll go with your suggestion, since you know more about it than I do.